docs/history.rst

```diff
@@ -1,6 +1,36 @@
 Change Log
 ==========
 
+2.2.0
+~~~~~
+* Minimum PROJ version is 6.1.0
+* `pyproj.crs` updates:
+    * Updated CRS repr (issue #264)
+    * Add Datum, CoordinateSystem, CoordinateOperation clases (issue #262)
+    * Added CRS mapping to/from CF 1.8 (pull #244)
```

Comment:
maybe spell out CF in full? (for those less familiar with the abbreviation)

Reply:
Good idea. Maybe a link as well.